frontend/src/pages/projects/screens/spawner/spawnerUtils.ts

```diff
@@ -376,3 +376,15 @@ export const checkRequiredFieldsForNotebookStart = (
     isDataConnectionValid
   );
 };
+
+export const isInvalidBYONImageStream = (imageStream: ImageStreamKind) => {
+  // there will be always only 1 tag in the spec for BYON images
+  // status tags could be more than one
+  const activeTag = imageStream.status?.tags?.find(
```

Comment:
You are getting the activeTag in several parts, we might want to convert this into a function right?